GitUI/Editor/FileViewerInternal.cs

```diff
@@ -168,50 +126,15 @@ void SetActiveLineNum(int line)
                 _diffHighlightService = DiffHighlightService.IsCombinedDiff(text)
                     ? CombinedDiffHighlightService.Instance
                     : DiffHighlightService.Instance;
-            }
-            else
-            {
-                TextEditor.ShowLineNumbers = true;
-                _lineNumbersControl.SetVisibility(false);
-            }
-
-            TextEditor.Text = text;
 
-            if (isDiff)
-            {
                 _lineNumbersControl.DisplayLineNumFor(text);
```

Comment:
This might be too early now: `_lineNumbersControl` holds a reference to the `TextEditor.ActiveTextAreaControl.TextArea` but `TextEditor.Text` has not been updated yet.
I'll check whether this matters.

Reply:
Please check, I grouped them together but the order may matter